docs/site/Decorators.md

```diff
@@ -208,7 +208,7 @@ You can find specific use cases in [Writing Controller methods](Controllers.md#w
 
 ### RequestBody Decorator
 
-Syntax: see [API documentation](https://github.com/strongloop/loopback-next/blob/0739ffcfe3ef50e0bfd86055c0f4e29fd6925be0/packages/openapi-v3/src/request-body-decorator.ts#L20-L79)
+Syntax: see [API documentation](https://github.com/strongloop/loopback-next/blob/0739ffcfe3ef50e0bfd86055c0f4e29fd6925be0/packages/openapi-v3/src/decorators/request-body.decorator.ts#L20-L79)
```

Comment:
Can we link to actual `apidocs` for this instead of a code.

Reply:
We don't have apidocs for this since the `@param` issue hasn't been fixed yet. If it has, it didn't work. Search for 'Describe an input parameter of a Controller method' (the description of the tsdoc for `@param`) and you won't get any hits.

I will point the link to this commit though. I meant to do it, but I guess I for got :p